docs/releasenotes/release-1.120.md

```diff
@@ -35,6 +35,7 @@ output fields from GCP APIs are in `status.observedState.*`
   * Added `pieChart` widgets.
   * Added `sectionHeader` widgets.
   * Added `singleViewGroup` widgets.
+  * Added `timeSeriesTable` widgets.
```

Comment:
I want to merge the PR without this change. Do you mind reverting?

Reply:
Okay, just checked that the `timeSeriesTable` field has been checked in. Thank you for updating this note!